packages/commonwealth/server/routes/createComment.ts

```diff
@@ -348,12 +323,12 @@ const createComment = async (
   emitNotifications(
     models,
     NotificationCategories.NewComment,
-    root_id,
+    thread_id,
     {
       created_at: new Date(),
-      root_id: id,
+      thread_id: thread_id,
       root_title,
-      root_type: prefix,
+      root_type: ProposalType.Thread,
```

Comment:
Can we remove this column altogether?

Reply:
It looks like we can, but I think this should be best left to another ticket, because it will require a refactor of a large unrelated portion to the root_id refactor.

If you still think it should be part of this PR let me know and I will go ahead with it as well.

Reply:
Can you make a ticket explaining the scope of other affected areas?

Reply:
On db side:
Notifications table needs to be changed. object_id should have root_type remove

On front end side quite a few components rely on this. From my IDE I can pick out:
reactionCounts.ts
notification_rows.ts
user_dashboard_row.tsx
user_dashboard_row_top.tsx
notificationFormatter.ts

backend:
/editComment
/createComment
/createReaction

and maybe others that are not immediately obvious to me.

Reply:
Got it. Please make a ticket with this info and we will mark this as resolved for this PR.